### ISSUE TITLE
Add <Skip to main> keyboard navigation aid

### ISSUE THREAD

**Issue by sigurdm** [dart-lang/pub-dev#7242]
It should be 
* Hidden by default
* the first element in the tab-order to receive keyboard focus
* Be a link to the first search result.

See the google search results page for inspiration.

**Comment by sigurdm** [comment]
Actually we could use this on all pages that define a `main` role.

This should probably be all pages
